src/validation.rs

```diff
@@ -263,6 +263,9 @@ pub(crate) fn validate(claims: ClaimsForValidation, options: &Validation) -> Res
     }
 
     match (claims.aud, options.aud.as_ref()) {
+        (TryParse::Parsed(_), None) => {
```

Comment:
Can you put the snippet of the RFC as a comment above here as well?

Reply:
```suggestion
        // Each principal intended to process the JWT MUST
        // identify itself with a value in the audience claim. If the principal
        // processing the claim does not identify itself with a value in the
        // "aud" claim when this claim is present, then the JWT MUST be
        //  rejected.
        (TryParse::Parsed(_), None) => {
```

Like this?

Reply:
Yep

Reply:
Done :)